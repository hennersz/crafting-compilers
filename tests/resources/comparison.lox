print 2 > 1;
print 1 > 2;
print 1 >= 1;
print 1 >= 2;
print 1 < 2;
print 2 < 1;
print 1 <= 1;
print 2 <= 1;
print 1 != 1;
print 1 != 2;
print !nil;
print nil == nil;
print nil == 1;