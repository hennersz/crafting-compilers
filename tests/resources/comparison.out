true
false
true
false
true
false
true
false
false
true
true
true
false